### PR TITLE
adding preview prefix when load app

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

in order to load the correct component we need to add the prefix /preview 

Fixes # (THEEDGE-3705)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Review by matoval — APPROVED** [review]

**Review by adamrdrew — COMMENTED** [review]
Based on the example in https://github.com/RedHatInsights/insights-dashboard/blob/master/config/dev.webpack.config.js#L8 this looks good to me!

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -50,6 +50,9 @@ const webpackProxy = {
 const deps = require('../package.json').dependencies;
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
+  appUrl: process.env.BETA ?  '/preview/edge' : '/edge',
+  deployment: process.env.BETA ? 'preview/apps' : 'apps',
```
Comment (omaciel) [inline config/dev.webpack.config.js:54]: Should it be `apps` or `/apps`?

File: config/prod.webpack.config.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const { resolve } = require('path');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
+  appUrl: process.env.BETA ?  '/preview/edge' : '/edge',
+  deployment: process.env.BETA ? 'preview/apps' : 'apps',
```
Comment (omaciel) [inline config/prod.webpack.config.js:5]: Same question as before

**Review by omaciel — COMMENTED** [review]
/lgtm

File: config/dev.webpack.config.js (hunk 1 above)
Comment (omaciel) [reply config/dev.webpack.config.js:54]: Looks like an existing example uses `apps`.  Marking it as resolved.

Comment (adamrdrew) [reply config/dev.webpack.config.js:54]: insights-dashboard has the deployment value with no leading slash https://github.com/RedHatInsights/insights-dashboard/blob/master/config/dev.webpack.config.js#L10 so I *think* its right but I don't actually know

**Review by omaciel — APPROVED** [review]
/lgtm

**Pull request merged**